requirements/test.in

```diff
@@ -1,11 +1,13 @@
 # Requirements for test runs
 
--r base.txt                         # Package dependencies
+-r base.in                         # Package dependencies
+-r travis.in
```

Comment:
Why was this needed here?  We usually only need the dependencies from that file in CI and dev environments.

Reply:
travis includes tox-battery, was having some issues locally that dependencies were not being updated in tox env properly so added travis requirements here.

Reply:
removed.